app/classes/Transvision/Strings.php

```diff
@@ -103,6 +103,32 @@ public static function multipleStringReplace($replacements, $string)
     }
 
     /**
+     * Highlight special characters in the string
+     *
+     * @param  string $string Source text
+     * @param  string $white_space Optional param to specify if we need to
```

Comment:
haha, yeah, I used a boolean initially then thought you were suggesting to use a string. Thanks, will update that. Yeah, $exclude_whitespaces should be better, since there’s a single call highlight them